internal/storage/postgres/flowerbed/flowerbed_test.go

```diff
@@ -313,3 +313,109 @@ func TestTreeClusterRepository_Archived(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestFlowerbedRepository_UnlinkSensorID(t *testing.T) {
+	t.Run("should unlink sensor ID successfully", func(t *testing.T) {
+		// given
+		suite.ResetDB(t)
+		suite.InsertSeed(t, "internal/storage/postgres/seed/test/flowerbed")
+		r := NewFlowerbedRepository(suite.Store, mappers)
+
+		// when
+		err := r.UnlinkSensorID(context.Background(), "1")
+
+		// then
+		assert.NoError(t, err)
+
+	})
+	t.Run("should return no error if sensor ID does not exist", func(t *testing.T) {
+		// given
+		suite.ResetDB(t)
+		r := NewFlowerbedRepository(suite.Store, mappers)
+
+		// when
+		err := r.UnlinkSensorID(context.Background(), "9999")
+
+		// then
+		assert.NoError(t, err)
+	})
+
+	t.Run("should return error when database is unreachable", func(t *testing.T) {
```

Comment:
`should return error when the context is canceled.` Something like that the test case is called in the other tests. Let's unify as much as we can 👍